pkg/cmd/application/create/app_create.go

```diff
@@ -302,10 +327,9 @@ func createPRWithUpdatedReposListForTenant(
 }
 
 func (opts *AppCreateOpt) createTenantInput(existingTenant []coretnt.Tenant) userio.InputSourceSwitch[string, *coretnt.Tenant] {
```

Comment:
Your implementation allows to pick root tenant, which should not be allowed for app creation. And it has the same bug I fix here: https://github.com/coreeng/corectl/pull/31/files

I'll remove the option to pick root tenant, because it's also should not be possible for the `p2p export` cmd (the only place where this func is used at the moment, as far as I see). Is it ok for you?

Reply:
yeap, cool, thanks

Reply:
Fixed